pkg/manager/member/tikv_member_manager.go

```diff
@@ -183,6 +188,12 @@ func (tkmm *tikvMemberManager) syncStatefulSetForTidbCluster(tc *v1alpha1.TidbCl
 		return err
 	}
 
+	if features.DefaultFeatureGate.Enabled(features.AutoScaling) {
+		if err := tkmm.syncTiKVAutoScalingConfigMap(tc); err != nil {
+			return err
+		}
+	}
+
```

Comment:
> In my understanding, this is an optional feature. I think it's fine to have some tidb version requirements, e.g. supporting setting hotRegion automatically after tidb 3.0.x or 4.0.0.

The replica of tikv configmap only for auto-scaling is controlled by featureGate, I think this met your requirement.
@cofyc